portfolio/src/main/java/com/google/sps/servlets/ListDataServlet.java

```diff
@@ -36,15 +39,20 @@
 @WebServlet("/list-comments")
 public class ListDataServlet extends HttpServlet {
 
+  // Only comments with sentiment score >= this value will be fetched and displayed.
+  private static double commentFilterThreshold = -0.5; 
```

Comment:
nit: use constant case for constants. See https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names